### ISSUE TITLE
layout-large should be replace by sw600dp

### ISSUE THREAD

**Issue by cloudbank** [codepath/android_guides#280]
layout-large is deprecated even though it is still in the android docs for flexible layouts.
It should say sw600dp for tablets and the like on:
https://github.com/codepath/android_guides/wiki/Flexible-User-Interfaces/_edit

**Comment by nesquena** [comment]
Thanks! I've updated the text on the page. Unfortunately, the entire guide screenshots are quite out of date (using eclipse from years ago), hopefully someone can help refresh them in the future.

**Comment by ultrasamad** [comment]
@nesquena  Yes. I was following along with Android studio but when I got to creating alternative layout for tablet mode, things seem totally different. So I couldn't continue. I will be nice if those screenshots are updated.
